tests/Wetcon.PactwarePlugin.OpcUaServer.Plugin.Tests/Wetcon.PactwarePlugin.OpcUaServer.Plugin.Tests.csproj

```diff
@@ -262,6 +262,9 @@
     <Reference Include="System.Xml.ReaderWriter, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <HintPath>..\..\packages\System.Xml.ReaderWriter.4.3.0\lib\net46\System.Xml.ReaderWriter.dll</HintPath>
     </Reference>
+    <Reference Include="Wetcon.IoLink.Helper">
+      <HintPath>..\..\src\Wetcon.PactwarePlugin.OpcUaServer.Plugin\DLLs\Wetcon.IoLink.Helper.dll</HintPath>
```

Comment:
I would prefer a `\lib` folder for binary dependencies like in https://gist.github.com/davidfowl/ed7564297c61fe9ab814 so we don't end up with multiple DLLs directories across projects.